### DISCUSSION TITLE
Is there some version with real costs from Azure provider?

### DISCUSSION THREAD

**Discussion by alexandrezanetti** [infracost/infracost#1986] [Q&A]
Hi!

I'm using Infracost V0.10.9 and I'm looking for some version with real costs from Azure provider.

Can someone help me about ?

**Comment by alikhajeh1** [comment]
@alexandrezanetti Infracost doesn't do that just now, help me understand why that's useful. Or what would you like to do with that info? The Azure portal cost management tools show the actual costs already
